browser/src/react-redux/App/Signup/SignupForm.jsx

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from "react";
+import axios from "axios";
+import Form from "./Form";
+import utility from "./signupUtility";
+
+class signupForm extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      password: "",
+      confirmPassword: "",
+      errors: {
+        confirmPassword: "",
+        password: "",
+        email: "",
+        general: ""
+      },
+      buttonDisabled: false
+    };
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
I would just use an arrow function with your methods below and bind `this` that way.